### PR TITLE
Added no-op augment_bounding_boxes() methods 

### PR DESCRIPTION
**Pull request by divyashreepathihalli**
Added no-op augment_bounding_boxes() methods to all KPLs that don't modify the bounding boxes

### CODE REVIEW

**Comment by LukeWood** [comment]
Thanks for the PR Divya!

File: keras_cv/layers/preprocessing/auto_contrast_test.py [hunk 1]
```diff
@@ -90,3 +90,13 @@ def test_auto_contrast_properly_converts_value_range(self):
 
         self.assertTrue(tf.math.reduce_any(ys[0] == 0.0))
         self.assertTrue(tf.math.reduce_any(ys[0] == 1.0))
+
+    def test_augment_bbox_dict_input(self):
```
Comment (LukeWood) [inline keras_cv/layers/preprocessing/auto_contrast_test.py:94]: Thanks for including all of these unit tests!

Can we create a single `with_bounding_boxes_test.py` file based on: https://github.com/keras-team/keras-cv/blob/master/keras_cv/layers/preprocessing/with_labels_test.py to handle all of these that do not interact with bounding_boxes?

**Comment by atuleu** [comment]
Is it not a duplicate of #588 ?

**Comment by LukeWood** [comment]
@atuleu looks great, good point. Ill check it out.

**Comment by LukeWood** [comment]
@divyashreepathihalli @atuleu lets remove the tests from THIS PR, and merge the behavior, then rely on #588 to create the test cases.

Sound like a plan?

**Comment by LukeWood** [comment]
LGTM, thanks @divyashreepathihalli

File: keras_cv/layers/preprocessing/with_bounding_boxes_test.py [hunk 2]
```diff
@@ -71,24 +71,20 @@
...
-    def test_can_run_with_labels(self, layer_cls, init_args):
+    @parameterized.named_parameters(*TEST_CONFIGURATIONS)
+    def test_can_run_with_boudning_boxes(self, layer_cls, init_args):
         layer = layer_cls(**init_args)
 
         img = tf.random.uniform(
             shape=(3, 512, 512, 3), minval=0, maxval=1, dtype=tf.float32
         )
-        labels = tf.ones((3,), dtype=tf.float32)
+        bounding_boxes = tf.random.uniform((3, 2, 4), 0, 255, dtype=tf.float32)
```
Comment (LukeWood) [inline keras_cv/layers/preprocessing/with_bounding_boxes_test.py:80]: bounding_boxes should be of shape (3, 2, 5), not (3, 2, 4).

class labels are important!

File: keras_cv/layers/preprocessing/with_bounding_boxes_test.py [hunk 3]
```diff
@@ -71,24 +71,20 @@
 
 
 class WithLabelsTest(tf.test.TestCase, parameterized.TestCase):
-    @parameterized.named_parameters(
-        *TEST_CONFIGURATIONS,
-        ("CutMix", preprocessing.CutMix, {}),
-    )
-    def test_can_run_with_labels(self, layer_cls, init_args):
+    @parameterized.named_parameters(*TEST_CONFIGURATIONS)
+    def test_can_run_with_boudning_boxes(self, layer_cls, init_args):
```
Comment (LukeWood) [inline keras_cv/layers/preprocessing/with_bounding_boxes_test.py]: test_can_run_with_boudning_boxes => test_can_run_with_bounding_boxes

File: keras_cv/layers/preprocessing/auto_contrast.py [hunk 4]
```diff
@@ -64,6 +64,9 @@ def augment_image(self, image, transformation=None, **kwargs):
         result = tf.where(tf.math.is_nan(result), original_image, result)
         return result
 
+    def augment_bounding_boxes(self, bounding_boxes, transformation, **kwargs):
```
Comment (LukeWood) [inline keras_cv/layers/preprocessing/auto_contrast.py:67]: drop transformation, it isn't used

Comment (divyashreepathihalli) [reply keras_cv/layers/preprocessing/auto_contrast.py:67]: I tried to keep it consistent the BaseImageAugmentationLayer call(). I'll update it.

**Comment by LukeWood** [comment]
looks like a unit test is failing

**Comment by divyashreepathihalli** [comment]
> 
l tried to remove it for all augment_* no op methods. but the labels one are throwing errors. I changed that back. It should be good now.

**Comment by LukeWood** [comment]
Thanks for trying that @divyashreepathihalli, we can do that as a follow up down the line.  Feel free to open a github issue - thanks again for this!

**Pull request merged**